hoodie-common/src/main/java/com/uber/hoodie/common/table/HoodieTimeline.java

```diff
@@ -105,6 +105,12 @@
    */
   HoodieTimeline filterCompletedAndCompactionInstants();
 
+  /**
+   * Filter this timeline to just include commits (commit/deltacommit) and compaction actions
+   * @return
+   */
+  HoodieTimeline filterCommitsAndCompactionTimeline();
```

Comment:
There is already a getCommitsAndCompactionTimeline in HoodieActiveTimeline, do we need this ?

Reply:
+1

Reply:
I need similar method in HoodieTimeline interface. I have renamed the method to getCommitsAndCompactionTimeline so that they will be overridden by HoodieDefaultTimeline and HoodieActiveTImeline.